### PR TITLE
Various improvements to runners

### PR DESCRIPTION
**Pull request by dannysepler**
Had some fun with this.

1) Is multiprocessing necessary? Not really. Is it fun? Absolutely!
2) Figured your parse_all may want to always setup the folder structures by default?

### CODE REVIEW

File: sources.json [hunk 1]
```diff
@@ -7,12 +7,12 @@
   "spain": {
     "primarySource": "https://github.com/datadista/datasets/tree/master/COVID%2019",
     "dataProvenance": "http://www.datadista.com",
-    "license": "unclear"
+    "license": "GNU Affero General Public License v3.0"
```
Comment (mserranom) [inline sources.json:10]: This is not correct, there's an ad-hoc license in the source readme

**Comment by mserranom** [comment]
- Shouldn't be merged with the errors in licenses
- Opinion: adding concurrency is including unnecessary complexity in the codebase.

**Comment by rneher** [comment]
Thank you, @dannysepler! these are useful improvements. But @mserranom points are well taken and keeping the code base simple will help others when adding more sources.

**Comment by dannysepler** [comment]
yeah, probably overkill at the moment! i'll close this out and come up with something more useful next time 👍

**Pull request closed without merging**